RELEASING.md

```diff
@@ -8,6 +8,3 @@ This repository is set up for autorelease. The process involves pushing a new gi
  1. `git commit -am "Prepare for release X.Y.Z."` (where X.Y.Z is the new version)
  1. `git tag -a X.Y.Z -m "Version X.Y.Z"` (where X.Y.Z is the new version)
  1. `git push && git push --tags`
- 1. Update the `gradle.properties` to the next SNAPSHOT version.
```

Comment:
@brennan wondering if this is required -- I think these last steps are not necessary with new auto release flow. what do you think?

Reply:
I agree they're not necessary. Feel free to remove! 🥇